assets/css/style.css

```diff
@@ -1,3 +1,276 @@
-.makeHeaderRed {
-  color: red;
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: calc(20px + 0.390625vw);
+}
+
+body {
+  background-color: black;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 18px;
+  background-color: #3e3c3c;
+  box-shadow: 1px 2px 10px black;
+}
+
+.logo {
+  margin-left: 3%;
+}
+
+.hamburgerMenu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+}
+
+.nav-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+  padding-right: 30%;
+}
+
+.list-item {
+  list-style: none;
+  margin: 0;
+  padding: 0 25px;
+}
+
+.list-item a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.list-item a:hover,
+button:hover {
+  transform: translate(4px, 2px);
+  box-shadow: #ff6b00;
+}
+
+.hamburger-icon {
+  display: none;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-right: 20px;
+}
+
+.hamburger-icon .bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #e5e5e5;
+  border-radius: 2px;
+}
+
+.main-area {
+  background-image: url("../images/Illustration-header.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  height: 100%;
+  font-family: 'Crete Round', serif;
+  margin-top: 4rem;
+}
+
+.main-header {
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-size: 33px;
+  margin-bottom: 1rem;
+}
```

Comment:
- [ ] For the h1 tag, I would recommend that you use the typography used in the Figma template. That includes the `font-size`, `line-height` and `font-size.` Using the same typography would make your design look identical to that of the Figma template that you chose. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219816178-6bad4d7a-2547-4ea9-9958-8ff592eda5dd.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/219816228-4d6f5efe-f724-460b-b792-c0b446fe6d97.png)

### CSS declarations

![image](https://user-images.githubusercontent.com/77106962/219816271-d156f216-7454-4ad1-bf57-8fe8fbd04e4d.png)